src/pages/resultsView/enrichments/AlterationEnrichmentsTable.tsx

```diff
@@ -145,7 +159,8 @@ export default class AlterationEnrichmentTable extends React.Component<IAlterati
         return (
             <AlterationEnrichmentTableComponent initialItemsPerPage={20} paginationProps={{ itemsPerPageOptions: [20] }}
                 columns={orderedColumns} data={this.props.data} initialSortColumn={this.props.initialSortColumn} 
-                onRowClick={this.props.onGeneNameClick ? this.onRowClick : undefined} dataStore={this.props.dataStore}/>
+                onRowClick={this.props.onGeneNameClick ? this.onRowClick : undefined} dataStore={this.props.dataStore}
```

Comment:
here is where the table component updates store state on click.  Where I've commented above, I think we should be reading the state from the store rather than from the visibleGenes property of the component.   It is a big antipattern for react components to read state from each other.  If state needs to be shared, it should be managed by a common store.  It would appear that the data already exists in the store.